src/smolagents/prompts/code_agent.yaml

```
@@ -150,8 +150,12 @@ system_prompt: |-
   Here is a list of the team members that you can call:
   ```python
   {%- for agent in managed_agents.values() %}
-  def {{ agent.name }}("Your query goes here.") -> str:
-      """{{ agent.description }}"""
+  def {{ agent.name }}(task: str) -> str:
```

You can add `additional_args` here to hint the manager that it can pass other args:
```suggestion
  def {{ agent.name }}(task: str, additional_args: dict[str, Any]) -> str:
```

---

I think I will treat that in a subsequent PR.